fpga/src/main/scala/arty/IOBinders.scala

```diff
@@ -35,4 +36,26 @@ class WithSPIFlashIOPassthrough  extends OverrideIOBinder({
     }).unzip
     (ports, cells2d.flatten)
   }
+})
+
+class WithGPIOPassthrough  extends OverrideIOBinder({
+  (system: HasPeripheryGPIOModuleImp) => {
+    val (ports: Seq[GPIOPortIO], cells2d) = system.gpio.zipWithIndex.map({ case (u, i) =>
+      val (port, ios) = IOCell.generateIOFromSignal(u, s"gpio_${i}", system.p(IOCellKey), abstractResetAsAsync = true)
+      (port, ios)
+    }).unzip
+
+    
+
+    for(iof_0 <- system.iof(0).get.iof_0){
+      iof_0.default()
+    }
+    for(iof_1 <- system.iof(0).get.iof_1){
+      iof_1.default()
+    }
+    val iof_0 = system.iof(0).get.iof_0
+    val iof_1 = system.iof(0).get.iof_1
+    (ports, cells2d.flatten)
+    
+  }
```

Comment:
Can we add this to the chipyard/IOBinders.scala instead? Since this code is not specific to arty.
This should also be named `WithGPIOIOCells` since this code does instantiate IOCells in the ChipTop.

Also for generality let's do
``` 
system.iof.map ({ i =>
  i.iof_0.default()
  i.iof_1.default()
})
```